library/src/test/java/com/okta/oidc/OktaResultFragmentTest.java

```diff
@@ -134,6 +139,29 @@ public void handleAuthorizationResponseLogoutSuccess() throws AuthorizationExcep
         assert (resultTypeCapture.getValue() == AuthenticationResultHandler.ResultType.SIGN_OUT);
     }
 
+    @Test
+    public void handleAuthorizationResponseLogoutSuccessWhenFragmentManagerIsDestroyed() throws AuthorizationException {
```

Comment:
This test doesn't fail in our master branch, which I'd expect it to given the change in production code.